src/tolerances.jl

```diff
@@ -13,5 +13,6 @@ source code in calls to the [`isapprox`](@ref) function:
 isapprox(a::T, b::T, atol=atol(T))
 ```
 """
+atol(::Type{T}) where T = zero(T)
```

Comment:
If this change is unrelated to the Point constructor, we should probably add it in a separate PR.

Reply:
Yes, this change should be in another PR. Thank you!